docs/rofl.md

```diff
@@ -240,6 +264,35 @@ offer. Specify the extension period with [`--term`][term-flags] and
 
 [term-flags]: #deploy
 
+### Show machine logs {#machine-logs}
+
+You can fetch logs from your running ROFL app using `oasis rofl machine logs`.
+
+```shell
+oasis rofl machine logs
```

Comment:
This should be moved as an example invocation to `../examples/rofl/machine-logs.in.static`.